.github/ci/deployments/common/k8s-relay-rollout.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps.cloudrobotics.com/v1alpha1
```

Comment:
I think we can drop all these rollout yaml files (deployments/common and deployments/robco-integration-test/kubernetes/). The would be referenced by the deploy.sh on the infrastructure folder in insrc.

Reply:
They are referenced from integration_test.sh part of this PR ("Deploy approllouts and configuration"), but maybe we can drop that and make that part of the apps CI. Would you prefer that?

Reply:
ahh lines 119-121. We certainly don't need prometheus, I am not sure if we rely for k8s relay for any of the tests. We can certainly drop the copy of the prometheus-rollout.yaml, we can cleanup the k8s-relay later after maybe testing it.

Reply:
The integration test tests that the relay can reach the simulated robot.